mantle/cmd/kola/options.go

```diff
@@ -58,7 +57,8 @@ func init() {
 	sv(&kola.Options.IgnitionVersion, "ignition-version", "", "Ignition version override: v2, v3")
 	ssv(&kola.BlacklistedTests, "blacklist-test", []string{}, "List of tests to blacklist")
 	bv(&kola.Options.SSHOnTestFailure, "ssh-on-test-failure", false, "SSH into a machine when tests fail")
-	sv(&kola.Options.CosaBuild, "cosa-build", "", "File path for coreos-assembler meta.json")
+	sv(&kola.Options.CosaWorkdir, "workdir", "", "coreos-assembler working directory")
+	sv(&kola.Options.CosaBuild, "build", "", "coreos-assembler build ID")
```

Comment:
Bikeshed but how about `kola.Options.CosaBuildId` to clarify what it is - we would use the `Build` term for the *parsed* thing.

Reply:
Done!